github/config.go

```diff
@@ -99,30 +65,25 @@ func (c *Config) Clients() (interface{}, error) {
 	}
 	v3client.BaseURL = uv3
 
-	org.v3client = v3client
-	org.v4client = v4client
+	owner.v3client = v3client
+	owner.v4client = v4client
 
-	if c.Individual {
-		org.name = ""
-	} else {
-		org.name = c.Organization
-
-		remoteOrg, _, err := org.v3client.Organizations.Get(ctx, org.name)
+	owner.name = c.Owner
+	if owner.name == "" {
+		// Discover authenticated user
+		user, _, err := owner.v3client.Users.Get(ctx, "")
 		if err != nil {
 			return nil, err
 		}
-		org.id = remoteOrg.GetID()
-	}
-
-	return &org, nil
-}
-
-func insecureHttpClient() *http.Client {
-	return &http.Client{
-		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{
-				InsecureSkipVerify: true,
-			},
-		},
+		owner.name = user.GetLogin()
+	} else {
+		remoteOrg, _, err := owner.v3client.Organizations.Get(ctx, owner.name)
+		if err == nil {
+			if remoteOrg != nil {
+				owner.id = remoteOrg.GetID()
+				owner.IsOrganization = true
+			}
+		}
 	}
+	return &owner, nil
 }
```

Comment:
this PR removed support for the `insecure` flag, but didn't remove the `insecure` flag from the provider's accepted `insecure` config, can we reinstate the behavior or remove the flag?